examples/Perl5/client.pl

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/perl5
+
+use JSON::XS 'decode_json';
+use ZMQ::FFI qw(ZMQ_SUB);
+use Time::HiRes q(usleep);
+use Compress::Zlib;
+
+my $endpoint = "tcp://eddn.edcd.io:9500";
+my $ctx      = ZMQ::FFI->new();
+
+my $s = $ctx->socket(ZMQ_SUB);
+
+$s->connect($endpoint);
+
+{
```

Comment:
is this nesting superfluous?

Reply:
Totally, I had no vimrc when writing this and indent totally went nuts when I tried to fix it :)